src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <VcardTemplate id="extension-request">
+    <template #icon>
+      mdi-calendar-range
+    </template>
+
+    <template #title>
+      Extension Request
+    </template>
+
+    <template #content>
+      <div class="px-6 py-7">
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <strong>Is this the first AGM?</strong>
+          </v-col>
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <v-radio-group
+              v-model="data.isFirstAgm"
+              class="mt-0 pt-0"
+            >
+              <v-radio
+                label="Yes"
+                :value="true"
+              />
+              <v-radio
+                label="No - Specify AGM Year below"
+                :value="false"
+              />
+            </v-radio-group>
+          </v-col>
+        </v-row>
+
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <strong>AGM Year</strong>
+          </v-col>
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <v-text-field
+              v-model="agmYear"
```

Comment:
Here's another way to enable data sync with the parent component.

Here, use what looks like a local variable....

Reply:
Obsolete. Resolving.